recipes/hist/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hist" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hist-{{ version }}.tar.gz
+  sha256: 48edff8f498ea7c5c6d8057c6f001a2ce394acd0923b921c56b76fb732e11df3
+
+build:
+  number: 0
+  skip: true   # [py2k]
```

Comment:
conda-forge doesn't support py2 by default, so I think we can get rid of this and make this `noarch: python` (but see comment on `typing-extensions`) and just change the `python` to `python >=3.6` under the `host` and `run` sections of `requirements`.